src/components/App.jsx

```diff
@@ -38,16 +36,14 @@ export default class App extends React.Component {
 			isAuthenticated: AuthStore.isAuthenticated(),
 			displayName: ''
```

Comment:
if you're not going to use `props` anywhere inside the constructor whats the need of passing it to `super()` as `super(props)`.
Just doing `super()` without `props` is fine